### DISCUSSION TITLE
Account deletion

### DISCUSSION THREAD

**Discussion by I2rys** [EddieHubCommunity/LinkFree#9700] [Q&A] [answered]
Hello, How do you delete your account in Biodrop? The ToS doesn't specify how.

**Comment by eddiejaoude** [comment, marked as answer]
We are sorry to see you go. There is an issue that someone is working on, to delete an account. Until that is finished we can disable your profile. Would you like me to do that?

If you don't mind sharing, I would like to know why you are deleting your BioDrop account? This will help us improve the Open Source project.

**Reply by I2rys** [reply]
Sure! You can disable my account. As for the reason on why I want to delete my BioDrop account is that I just want to lessen my accounts on platforms to have a comprehensive control to my data. Thank you!

**Reply by eddiejaoude** [reply]
Thank you for explaining 👍 I have now disabled your account https://www.biodrop.io/I2rys

**Comment by harisdev-netizen** [comment]
@eddiejaoude kindly remove my account as well for now.

**Reply by eddiejaoude** [reply]
Please stop spamming me, I received the notifications on the discussion you created here https://github.com/EddieHubCommunity/BioDrop/discussions/10168

**Comment by nicfab** [comment]
Please delete my account also now.

**Reply by eddiejaoude** [reply]
Sure - all accounts will be deleted on the 10th of June

**Reply by nicfab** [reply]
I ask you to delete my account now.
